### PR TITLE
add a br tag

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request adds a `<br />` tag to add space for actions 

### How should this be reviewed?
...

### Any background context you want to provide?
Jen noticed a missing space while qa'ing.
add br tag to break 
Screen Shot 2020-07-01 at 2 29 05 PM

...

### Relevant tickets

References [173152297](https://www.pivotaltracker.com/n/projects/2401401/stories/173152297).

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ x ] Added screenshots of front-end changes on small, medium, and large screens.

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/Badges/BadgesTab.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ const badgeModalContent = {
     unearnedText: (
       <span>
         Unlock this badge by
+        <br />
         {exploreCampaignsLink('signing up for a campaign')}.
```
Comment (mendelB) [inline resources/assets/components/pages/AccountPage/Badges/BadgesTab.js:60]: I wonder if we can achieve a space without a line break by adding a literal space string {' '}?. (Something I've seen Prettier formatter do for instances [like this,](https://github.com/DoSomething/phoenix-next/blob/4c20abeab4f7e00177ae60d07b54763892a6b966/resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js#L169-L173) but I suppose here they couldn't catch it).
```suggestion
        Unlock this badge by{' '}{exploreCampaignsLink('signing up for a campaign')}.
```

Comment (Dinnall) [reply resources/assets/components/pages/AccountPage/Badges/BadgesTab.js]: That's an interesting approach. I just added a new commit to reflect this.
I would have prefered to turn the entire `unearnedText` into a template literal, but that wouldn't work here

Comment (mendelB) [reply resources/assets/components/pages/AccountPage/Badges/BadgesTab.js]: Nice! And confirming that it's showing up correctly for both badges now? 

The reason I think this is neat is that it gives us a space as opposed to a line break which might not be applicable across all viewport sizes. Another option I've used is to add an HTML space code to ensure the space is rendered `&nbsp;`

Comment (Dinnall) [reply resources/assets/components/pages/AccountPage/Badges/BadgesTab.js]: Super odd, `{" "}` worked but `&nbsp;` looked better, so I substituted it.

**Pull request merged**